site/_posts/2020-09-03-oidc-auth-api-usage.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Using the API with OpenID-Connect Authentication"
+author: jvlcek
+date: 2020-09-03
+comments: true
+published: true
+tags: tutorials
+---
+
+The goal of this blog post is to provide a basic understanding of how to use the ManageIQ REST API when configured with OpenID-Connect authentication.
+
+All the content in this blog post is only relevant when ManageIQ is configured for OpenID-Connect Authentication.
+
+**Contents:**
+---------------------------------------------------------------------
+
+- [**Configure ManageIQ For OpenID-Connect Authentication**](#configure-manageiq-for-openid-connect-authentication)
+- [**Using The ManageIQ REST API**](#using-the-manageiq-rest-api)
+- [**The OpenID-Connect REST API Authentication Modes**](#the-openid-connect-rest-api-authentication-modes)
+- [**Examples**](#examples)
+  - [Example: Using a JWT Token](#example:-using-a-jwt-token)
+  - [Example: Using a ManageIQ API Auth Token](#example:-using-a-manageiq-api-auth-token)
+  - [Example: Using Basic Auth For Admin](#example:-using-basic-auth-for-admin)
+
+# Configure ManageIQ For OpenID-Connect Authentication
+---------------------------------------------------------------------
+
+ManageIQ uses Apache’s <em>mod_auth_openidc</em> module to support OpenID-Connect authentication.
+The instructions for configuring Apache’s <em>mod_auth_openidc</em> for ManageIQ	differ for the
```

Comment:
nit-pick, spaces before differ

Reply:
That was a TAB that should be a space! Good catch!